main/urls.py

```diff
@@ -1,7 +1,12 @@
+# general imports
 from django.urls import path
 from main.views import home
 
+# api imports
```

Comment:
Sectioning using comments was the only way I found in order to avoid conflicts beetween `rest` and `push-notifications` branches

Reply:
Ok good, but then we would need to think about a better way to manage the different optional features.